internal/worker/server.go

```diff
@@ -97,7 +97,7 @@ func (s *Server) JobStatus(id uuid.UUID) (*JobStatus, error) {
 	if canceled {
 		state = common.CFailed
 	} else if !finished.IsZero() {
-		if result.OSBuildOutput.Success {
+		if result.OSBuildOutput != nil && result.OSBuildOutput.Success {
```

Comment:
Hm, as discussed on the other PR, this seems to be masking some underlying problem. I'd have loved to figure out the underlying issue there was. Is this a blocker for the rest of this PR? (My concern would be that even if this avoids us from crashing, we are still in an undefined/inconsistent state that we should probably have caught/handled elsewhere).

Reply:
Depending on the state of the system it fails. On a clean install I think it is fine -- but in my opinion if there's a pointer involved there is always going to be the potential for it to be nil and that should be checked, or the pointer should be eliminated.
I can drop this from this PR and we can cover it in the other one, I added it here so I could run the tests in my VM.

Reply:
If it doesn't block this, then let's continue the discussion on the other PR